src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -3,36 +3,61 @@
 [data-store-breadcrumb] {
   width: 100%;
 
+  @include media(">=notebook") {
+    width: 50%;
+  }
+
   [data-breadcrumb-list] {
     display: flex;
     align-items: center;
+    margin-left: calc(var(--fs-spacing-1) * -1);
     overflow-x: hidden;
 
-    li {
+    [data-breadcrumb-list-item] {
       display: inline-flex;
+      align-items: center;
+      padding: calc(var(--fs-spacing-0) / 2);
```

Comment:
I noticed a minor visual detail; the focus style is slightly trimmed. To solve it, we can increase the padding and reduce the margin.

![trimming-focus](http://g.recordit.co/M3FlPVEDsv.gif)

Reply:
Great!